tests/e2e/tests/critical-path/browser/filtering.e2e.ts

```diff
@@ -1,69 +1,66 @@
-import { addNewStandaloneDatabase } from '../../../helpers/database';
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
 import {
-    MyRedisDatabasePage,
     BrowserPage,
-    UserAgreementPage,
-    CliPage,
-    AddRedisDatabasePage
+    CliPage
 } from '../../../pageObjects';
 import {
     commonUrl,
     ossStandaloneConfig
 } from '../../../helpers/conf';
 import { COMMANDS_TO_CREATE_KEY, KeyTypesTexts } from '../../../helpers/constants';
 import { keyTypes } from '../../../helpers/keys';
+import { Chance } from 'chance';
 
-const myRedisDatabasePage = new MyRedisDatabasePage();
 const browserPage = new BrowserPage();
-const userAgreementPage = new UserAgreementPage();
-const addRedisDatabasePage = new AddRedisDatabasePage();
 const cliPage = new CliPage();
+const chance = new Chance();
+
+let keyName = chance.word({ length: 10 });
```

Comment:
Should be `const`. I didn't see you change it over the test

Reply:
its used in the line 33 and 49